### PR TITLE
definitions and comments in seq files

### PR DESCRIPTION
**Pull request by kherz**
suggestions for definitions and comments in seq-file header

### CODE REVIEW

File: seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.m
Comment (KerstinHut) [inline seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.m:2]: I'd probably put a short functionality description in front of this, just in case these files end up somewhere out of context. 
```suggestion
% Creates a sequence file for an APTw protocol with Sinc-Gaussian pulses, 90% DC and tsat of 2 s
```

File: seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.seq
Comment (KerstinHut) [inline seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.seq:2]:
```suggestion
# Created with MATLAB mr toolbox
```

**Review by KerstinHut — COMMENTED** [review]
Right now our definitions are all lower case but I propose we could write a conversion of our definintion parameters into the function that writes them into the seq file, so the definitions are uniform in the end?

**Comment by schuenke** [comment]
It's probably easier if we always define/use lower case parameter names! Would that be okay for you as well @kherz and @mzaiss ?

File: seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.seq
Comment (kherz) [reply seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.seq:2]: That's the standard line from the original pulseq matlab package, wouldn't change that

**Comment by mzaiss** [comment]
In the commitee we agreed on e.g. uppercase for periods Trec and Tsat similar to TR TE and lowercase for inter pulse train times tp, td.
Thus lets stick to case sensitive in general.

**Comment by kherz** [comment]
If you agree to that , I would make the changes for the other protocols as well

**Comment by KerstinHut** [comment]
I think uppercase is also fine since they are all distinguishable, but we will have to decide whether to use it in the actual python code as well or adjust to PEP8 and auto-change them in the seq-files. Pro PEP8: it would make the format of the code files compatible to other python projects and the coding more natural to python users, Con: the write_seq files aren't as uniform (and it might confuse someone who wants to switch languages)

**Comment by schuenke** [comment]
Okay, than lets keep it as it is. In python we will stick with the PEP8 style guide and work with all lowercase variables, but we will convert the variable names to the current style before writing the definitions.  We can easily include this name converter into the function that writes all parameters from the `seq_defs` struct/dict into the seq-file definitions.

**Review by schuenke — APPROVED** [review]
Fine for me now!

**Comment by mzaiss** [comment]
when this is switched back again 
"# Created by MATLAB mr toolbox"
This is as well finished, right?

**Comment by kherz** [comment]
Yes, just waiting for your approval. The line you mentioned is not changed in the commit

**Review by mzaiss — APPROVED** [review]

**Pull request merged**
